src/frontend/src/components/anchorInput.ts

```diff
@@ -86,21 +92,21 @@ export const mkAnchorInput = (props: {
 
   // Select the input when the component is rendered (and value is empty)
   const selectInput = (elem: Element): void => {
-    if (elem instanceof HTMLInputElement && elem.value === "") {
+    if (elem instanceof HTMLInputElement) {
       elem.select();
     }
   };
 
-  const template = html` <div class="l-stack c-input--anchor">
+  const template = html` <div class="c-input--anchor">
     <label class="c-input--anchor__wrap" aria-label="Identity Anchor">
       <input
         ${ref(userNumberInput)}
-        ${mount(selectInput)}
+        ${focus ? mount(selectInput) : ""}
```

Comment:
not sure with lit but in React or Stencil when nothing need to be rendered commonly `undefined` (rather than `""`) is used. 

would `${focus ? mount(selectInput) : ""}` work?

or even more fancy `${focus && mount(selectInput)}`?

Reply:
> when nothing need to be rendered commonly undefined (rather than "") is used.

Mh, good question

> would `${focus ? mount(selectInput) : ""}` work?

That's what's there now right? 

> more fancy `${focus && mount(selectInput)}`?

Linter will definitely complain about that (mixing booleans and falsy/truthy)


---

Actually looks like `undefined`, `null` and `""` are all ok: https://lit.dev/docs/v1/lit-html/writing-templates/#rendering-nothing

> Sometimes, you may want to render nothing at all. The values undefined, null and the empty string ('') in a text binding all render an empty text node.